src/components/Shared/TableOfContentLink.tsx

```diff
@@ -0,0 +1,61 @@
+import { VariantProps, cva } from "class-variance-authority";
+import { ComponentPropsWithoutRef, MouseEventHandler } from "react";
+
+const style = cva(["block", "py-sm", ["hover:bg-[#F2F6FF]"]], {
+  variants: {
+    isActive: {
+      true: ["text-[#0052FF]", "shadow-[inset_4px_0px_0px]"],
```

Comment:
I don't know why I used custom box-shadows, probably we should update with the one from Spark tokens